src/quo2/components/avatars/channel_avatar.cljs

```diff
@@ -40,6 +40,6 @@
           {:color           (if dark?
                               colors/neutral-40
                               colors/neutral-50)
-           :container-style {:width  16
-                             :height 16}
+           :container-style {:width  12
```

Comment:
this lock icon was not matching the Figma designs correctly so I adjusted for both sizes and it now seems correct.